esp-backtrace/src/riscv.rs

```diff
@@ -15,6 +15,7 @@ pub(super) const RA_OFFSET: usize = 4;
 #[derive(Default, Clone, Copy)]
 #[cfg_attr(feature = "defmt", derive(defmt::Format))]
 #[repr(C)]
+#[allow(dead_code)]
```

Comment:
I'd do https://github.com/esp-rs/esp-hal/pull/2034/files#diff-7a0430174918ac5644e44736369b62b0298ea3c14c94e8fe409afd4af77786c6R18 instead, what do you think?

Reply:
Much better :+1:

Reply:
You'll have to gate the Debug implementation too :)